test/WebJobs.Script.Tests/Rpc/WorkerConfigFactoryTests.cs

```diff
@@ -194,7 +202,8 @@ public void AddArgumentsFromAppSettings_JavaOpts(string expectedArgument, string
             var config = configBuilder.Build();
             var scriptSettingsManager = new ScriptSettingsManager(config);
             var testLogger = new TestLogger("test");
-            var configFactory = new WorkerConfigFactory(config, testLogger);
+            var testSysRuntimeInfo = new TestSystemRuntimeInformation();
```

Comment:
as rest of the tests do not need reinitializing TestSystemRuntimeInformation, move this to constructor